teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, parent_permission = Trrue, name = 'Unknown')
```

Comment:
- [ OPTIONAL ] I think you have a syntax error here since it should be true not Trrue so kindly fix it ⭐